src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -298,6 +298,24 @@ private bool DerivesFrom(string typea, string typeb, ProtoCore.Core core)
             }
         }
 
+        /// <summary>
+        /// For checking the ML Context menu item
+        /// </summary>
+        public bool IsML
+        {
+            get { return dynamoViewModel.PreferenceSettings.DefaultNodeAutocompleteSuggestion == Models.NodeAutocompleteSuggestion.MLRecommendation; }
+            set { }
```

Comment:
we do not need setter in this case

Reply:
yh I thought the same but it threw me this: "A TwoWay or OneWayToSource binding cannot work on the read-only property"